More information added about "recipe-id-to-remove".

TimLiz opened the pull request:
Closes #832 (My own LOL)

## Changes proposed:
I added example how to use it

Merge please :D

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/ldtteam/MinecoloniesWiki?pullRequest=833) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/ldtteam/MinecoloniesWiki?pullRequest=833) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/ldtteam/MinecoloniesWiki?pullRequest=833) it.</sub>

anonymous563 commented:
I'm not sure what this adds to the page. Can you elaborate?

TimLiz commented:
> I'm not sure what this adds to the page. Can you elaborate? E.g., why add to this particular line but not all the others? Is this particular recipe tag especially unintuitive or different from the others?

I added the information to this line because this is the line responsible for 'recipe-id-to-remove', I added this at the request of someone from support on discord

MotionlessTrain commented:
That "someone" would be me :D
The reason of this change is that it is not clear at all what the "id" of a recipe is. I had a hard time figuring out myself which of those would be the id of the plate armour chest plate:
- `minecolonies:crafterrecipes/blacksmith/platearmorchestplate` (the complete path of the recipe in the datapack)
- `minecolonies:blacksmith/platearmorchestplate` (this turned out to be the actual id, after trying a few things)
- `minecolonies:platearmorchestplate` (just the name of the json file itself. This was used in the folder crafterrecipes/blacksmith, so that part could have been inferred)
I thought, in order to save the time to find out what the needed id for that property was, it would be useful to have an example on the wiki
